demo/send/mailingLists/7dc7a9bd-0a14-427f-894a-66d8f7a938db.json

```diff
@@ -0,0 +1 @@
+{"Campaign":null,"SendToDetails":null,"Content":null,"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":5,"MailingList":{"Id":"7dc7a9bd-0a14-427f-894a-66d8f7a938db","MailingListCode":"2930539","Name":"first mailing list","MailingListName":"first mailing list","ActiveMemberCount":5,"BouncedMemberCount":0,"RemovedMemberCount":0,"UnsubscribedMemberCount":0,"Status":0,"CreatedBy":"7dc7a9bd0a14427f894a66d8f7a938db","CreatedOn":"/Date(1641564378217-0000)/","UpdatedOn":"/Date(1658417249577-0000)/","DeletedMemberCount":0,"IsSupressionList":false},"Entity":null}
```

Comment:
is this "first mailing list" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?